db/migrate/20150420223003_create_admins.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAdmins < ActiveRecord::Migration
+  def change
+    create_table :admins do |t|
+      t.string :username
+      t.string :password
```

Comment:
I'll be working on authentication today to use bcrypt to encrypt the admin password in our database.  That's a separate chore though.